runtime/lua/vim/lsp.lua

```diff
@@ -60,6 +62,7 @@ lsp._request_name_to_capability = {
 ---
 --@param {...} (List of strings) List to write to the buffer
 local function err_message(...)
+  lsp.notifier(4, table.concat(vim.tbl_flatten{...}))
   nvim_err_writeln(table.concat(vim.tbl_flatten{...}))
```

Comment:
If we're already invoking the notifier, do we have to also invoke nvim_err_writeln?  That will write two error messages

Reply:
I was using this in my test but removed it to reduce the scope of the PR.